src/containerapp/azext_containerapp/_params.py

```diff
@@ -176,6 +176,12 @@ def load_arguments(self, _):
     with self.argument_context('containerapp env show') as c:
         c.argument('name', name_type, help='Name of the Container Apps Environment.')
 
+    with self.argument_context('containerapp env certificate create') as c:
+        c.argument('hostname', options_list=['--hostname'], help='The custom domain name.')
```

Comment:
Please do not explicitly define the default option name
```suggestion
        c.argument('hostname', help='The custom domain name.')
```

Reply:
The default option name for hostname is `--custom-domain-dns-suffix`/`--dns-suffix`, which is for environment-level custom domain.
This hostname here is for app-level hostname, which is the same concept as `--hostname` in `containerapp hostname` commands.

Reply:
> The default option name for hostname is --custom-domain-dns-suffix/--dns-suffix, which is for environment-level custom domain.

Sorry, I don't get what you mean. Could you please show me the relevant code links or explain the reasons?

Reply:
Sure!
In this `containerapp env certificate create` command, if we don't explicitly define hostname, it will inherit the definition from `containerapp env`, which will be `--custom-domain-dns-suffix` or `--dns-suffix`, and it means "The DNS suffix for the **environment**\'s custom domain.":
https://github.com/Azure/azure-cli-extensions/blob/2a4fceb2302cc9f0ff13e002ed8a7e2fc11de35f/src/containerapp/azext_containerapp/_params.py#L161-L162
But the hostname here in `containerapp env certificate create` is actually a different concept. It stands for the **app**-level custom domain. This concept is also used in `containerapp hostname` commands:
https://github.com/Azure/azure-cli-extensions/blob/2a4fceb2302cc9f0ff13e002ed8a7e2fc11de35f/src/containerapp/azext_containerapp/_params.py#L364-L365
So we want to use the name `--hostname` here to keep it consistent with `containerapp hostname` commands.

Reply:
Got it, thanks for your detailed explanation

Reply:
Np! Thanks for approving the PR!